Jenkinsfile

```diff
@@ -46,7 +46,7 @@ def runStages() {
 
     scmVars = checkout scm
 
-    if (currentBuild.currentResult == "SUCCESS" && env.CHANGE_TARGET == "stable" && env.CHANGE_ID) {
+    if (currentBuild.currentResult == "SUCCESS" && env.CHANGE_TARGET == "master" && env.CHANGE_ID) {
```

Comment:
Could we get it to run on master and stable PRs for now while we still manually promote the backend to stable?

Reply:
sure thing, I added a conditional for both branch names